packages/langium/src/utils/errors.ts

```diff
@@ -0,0 +1,17 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { CstNode } from '..';
```

Comment:
Minor: Please don't import from the index (could easily lead to cyclic dependency issues), but from the actual file.